Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/ClientGameNetPortal.cs

```diff
@@ -179,7 +192,7 @@ public static void StartClient(GameNetPortal portal, string ipaddress, int port)
         /// </remarks>
         /// <param name="portal"> </param>
         /// <param name="roomKey">The room name of the host to connect to.</param>
-        public static bool StartClientRelayMode(GameNetPortal portal, string roomKey, out string failMessage, CancellationToken cancellationToken)
+        public bool StartClientRelayMode(GameNetPortal portal, string roomKey, out string failMessage, CancellationToken cancellationToken)
```

Comment:
Since this is no longer used, should we remove it? Or do we intend to add another way to use this connection method in the future?